central/telemetry/centralclient/instance_config.go

```diff
@@ -0,0 +1,89 @@
+package centralclient
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/installation/store"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	tenantIDLabel = "rhacs.redhat.com/tenant"
```

Comment:
nit: just wondering about this value: would it be better to define this within `pkg/telemetry` somewhere, so that we can later on re-use it within the fleetshard-synchronizer and not have to manage the label name in two different places?